config/region.go

```diff
@@ -50,27 +51,41 @@ func EnsureRegionsCache(c Config, opts GetRegionOpts) (RegionsCache, error) {
 		return regionsCache, nil
 	}
 
-	token := &auth.UserToken{Token: opts.Token}
-	if token.Token == "" {
-		auth := &CliAuthenticator{}
-		_, token, err = auth.LoadAuth()
+	var client *scalingo.Client
+	if opts.SkipAuth {
+		client, err = ScalingoUnauthenticatedAuthClient()
 		if err != nil {
-			return RegionsCache{}, errgo.Notef(err, "fail to load authentication")
+			return RegionsCache{}, errgo.Notef(err, "fail to create an unauthenticated client")
+		}
+	} else {
+		token := &auth.UserToken{Token: opts.Token}
+		if token.Token == "" {
+			auth := &CliAuthenticator{}
+			_, token, err = auth.LoadAuth()
+			if err != nil {
+				return RegionsCache{}, errgo.Notef(err, "fail to load authentication")
+			}
 		}
-	}
 
-	debug.Println("[Regions] Get the list of regions to fill the cache")
-	client, err := ScalingoAuthClientFromToken(token.Token)
-	if err != nil {
-		return RegionsCache{}, errgo.Notef(err, "fail to create an authenticated Scalingo client using the API token")
+		debug.Println("[Regions] Get the list of regions to fill the cache")
+		client, err = ScalingoAuthClientFromToken(token.Token)
+		if err != nil {
+			return RegionsCache{}, errgo.Notef(err, "fail to create an authenticated Scalingo client using the API token")
+		}
 	}
+
 	regions, err := client.RegionsList()
 	if err != nil {
 		return RegionsCache{}, errgo.Notef(err, "fail to list available regions")
 	}
 
 	regionsCache.Regions = regions
 	regionsCache.ExpireAt = time.Now().Add(10 * time.Minute)
+
+	if opts.SkipAuth {
+		return regionsCache, nil
```

Comment:
Why don't you write the regions cache in case you skip the auth? I have my idea on this matter but it might be interesting to explain it in a comment here.